distributed-prover/src/main.rs

```diff
@@ -25,17 +25,17 @@ fn main() {
 
     match command {
         Command::Setup => {
-            circuits::keccak::setup();
```

Comment:
Instead of replacing keccak with EVM, we should add a nested command for circuit type.

Sample commands would be:

`./run.sh keccak setup`
`./run.sh evm prove 0`